### PR TITLE
Better emails

### PR DESCRIPTION
**Pull request by timiwahalahti**
This PR adds metabox where user can change form header, subject and content of email. All form fields can be used everywhere and there's also few global details which can be used; submission-id, referrer, form-title, form-id, user-id, timestamp, datetime, language, all-form-data.

Test, translate and update documentation before merge.

In a way, relates to #37.

### CODE REVIEW

**Comment by timiwahalahti** [comment]
Fixed build errors and handling of values which are arrays.

Previously fields with array values got saved wrongly because those were put throught `esc_html` and `wp_json_encode`. Removed these because `add_post_meta` does serialization and added `maybe_unserialize` with `implode`.

Also introduced three new filters to change glue for implode:
`wplf_email_array_field_implode_glue`
`wplf_{$form->post_name}_email_array_field_implode_glue`
`wplf_{$form->ID}_email_array_field_implode_glue`

File: classes/class-cpt-wplf-form.php [hunk 1]
```diff
@@ -169,16 +169,16 @@ function default_content_cpt( $content ) {
       // default content starts here:
       // @codingStandardsIgnoreStart
 ?>
-<p><label for="name"><?php esc_html_e( 'Please enter your name', 'wp-libre-form' ); ?></label>
+<p><label for="name"><?php esc_htmlesc_html_e( 'Please enter your name', 'wp-libre-form' ); ?></label>
```
Comment (anttiviljami) [inline classes/class-cpt-wplf-form.php]: here

**Comment by anttiviljami** [comment]
Brilliant stuff! Maybe we should pre-populate the email content with some sensible template containing `%all-form-fields%` ? Right now it's a bit too easy to accidentally save an empty content for the email

**Comment by anttiviljami** [comment]
No wait. :) That's exactly what it already does. Nevermind. 

I'm just gonna go ahead and merge.

Thank you so much!

**Comment by sparkalow** [comment]
Is this feature released?

**Comment by k1sul1** [comment]
Nope, the last release was released on Oct 24 and this was merged on Oct 26.

But we try to not break the master branch, so you can use that.

@anttiviljami @timiwahalahti should we do a release in the near future? There's bugfixes, new features and improvements. I think. GitHub kinda makes it hard to check that what does master contain that isn't in a release.

**Pull request merged**